README.md

```diff
@@ -2,20 +2,18 @@
 
 # Project Bicep - an ARM DSL
 
-***Note:** Bicep is currently an **experimental language** and we expect to ship breaking changes in future releases. It is not yet recommended for production usage. Please take a look at the [known limitations](#known-limitations) prior to opening any issues.*
-
 ## What is Bicep?
 
-Bicep is a Domain Specific Language (DSL) for deploying Azure resources declaratively. It aims to drastically simplify the authoring experience with a cleaner syntax and better support for modularity and code re-use. Bicep is a **transparent abstraction** over ARM and ARM templates, which means anything that can be done in an ARM Template can be done in bicep (outside of temporary [known limitations](#known-limitations)). All resource `types`, `apiVersions`, and `properties` that are valid in an ARM template are equally valid in Bicep on day one.
+Bicep is a Domain Specific Language (DSL) for deploying Azure resources declaratively. It aims to drastically simplify the authoring experience with a cleaner syntax, improved type safety, and better support for modularity and code re-use. Bicep is a **transparent abstraction** over ARM and ARM templates, which means anything that can be done in an ARM Template can be done in bicep (outside of temporary [known limitations](#known-limitations)). All resource `types`, `apiVersions`, and `properties` that are valid in an ARM template are equally valid in Bicep on day one.
 
-Bicep compiles down to standard ARM Template JSON files, which means the ARM JSON is effectively being treated as an Intermediate Language (IL).
+Bicep code is transpiled to standard ARM Template JSON files, which effectively treats the ARM Template as an Intermediate Language (IL).
 
 ## Goals
 
 1. Build the best possible language for describing, validating, and deploying infrastructure to Azure.
-1. The language should provide a *transparent abstraction* for the underlying platform. There must be no "onboarding step" to enable it to a new resource `type` and/or `apiVersion` in Bicep.
+1. The language should provide a *transparent abstraction* for the underlying platform. There must be no "onboarding step" to enable bicep to support for a new resource `type` and/or `apiVersion`.
```

Comment:
Can you capitalize "bicep" in the added text?

Reply:
+1, and there are quite a few other places this is missing

Reply:
I looked at so many instances of the word bicep that it now always looks like it is spelled wrong and my head hurts :)